packages/valory/protocols/ipfs/protocol-spec.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
This should be moved into `README.md` as this is currently how our generate all logic handles it. See open-aea